controllers/articles.controllers.js

```diff
@@ -42,4 +43,15 @@ exports.getArticles = (request, response, next) => {
         console.log(error)
         next(error)
     })
-}
\ No newline at end of file
+};
+
+exports.getCommentsByArticleId = (request, response, next) => {
+    const { article_id } = request.params; 
+    const promises = [fetchArticleById(article_id)]
+    if(article_id) {
```

Comment:
Do we really need this? I would think that if we are at this point, then the endpoint would already have assumed that we have an article_id? I may just be thinking about this wrong, but try just having your promise array include the fetchCommentsByArticleId function call always and see if that works. If not then ignore me and merge it :')

Reply:
ooh yes, you are correct: I popped both funcs in when initialising the promises array and got rid of the if statement and it works with less code. Thank you!